wpilibc/src/main/native/include/frc/PneumaticsBase.h

```diff
@@ -22,59 +22,217 @@ class PneumaticsBase {
+   * supported by the REV PH with the REV Analog Pressure Sensor connected to
+   * analog channel 0.
+   *
+   * The compressor will turn on when \a both:
+   *
+   * - The digital pressure switch indicates the system is not full AND
+   * - The analog pressure sensor indicates that the pressure in the system
+   * is below the specified minimum pressure.
+   *
+   * The compressor will turn off when \a either:
+   *
+   * - The digital pressure switch is disconnected or indicates that the system
+   * is full OR
+   * - The pressure detected by the analog sensor is greater than the specified
+   * maximum pressure.
+   *
+   * On CTRE PCM, this will enable digital control.
+   *
+   * @param minPressure The minimum pressure. The compressor will turn on
+   * when the pressure drops below this value and the pressure switch indicates
+   * that the system is not full.
+   * @param maxPressure The maximum pressure. The compressor will turn
+   * off when the pressure reaches this value or the pressure switch is
+   * disconnected or indicates that the system is full.
+   */
   virtual void EnableCompressorHybrid(
       units::pounds_per_square_inch_t minPressure,
       units::pounds_per_square_inch_t maxPressure) = 0;
 
+  /**
+   * Returns the active compressor configuration.
+   *
+   * @return The active compressor configuration.
+   */
   virtual CompressorConfigType GetCompressorConfigType() const = 0;
 
+  /**
+   * Sets solenoids on a pneumatics module.
+   *
+   * @param mask mask
+   * @param values values
+   */
   virtual void SetSolenoids(int mask, int values) = 0;
 
+  /**
+   * Gets a bitmask of solenoid values.
+   *
+   * @return values
+   */
   virtual int GetSolenoids() const = 0;
 
+  /**
+   * Get module number for this module.
+   *
+   * @return module number
+   */
   virtual int GetModuleNumber() const = 0;
 
+  /**
+   * Get a bitmask of disabled solenoids.
+   *
+   * @return bitmask of disabled solenoids
+   */
   virtual int GetSolenoidDisabledList() const = 0;
 
+  /**
+   * Fire a single solenoid shot.
+   *
+   * @param index solenoid index
+   */
   virtual void FireOneShot(int index) = 0;
 
+  /**
+   * Set the duration for a single solenoid shot.
+   *
+   * @param index solenoid index
+   * @param duration shot duration
+   */
   virtual void SetOneShotDuration(int index, units::second_t duration) = 0;
 
+  /**
+   * Check if a solenoid channel is valid.
+   *
+   * @param channel Channel to check
+   * @return True if channel exists
+   */
   virtual bool CheckSolenoidChannel(int channel) const = 0;
 
+  /**
+   * Check to see if the masked solenoids can be reserved, and if not reserve
+   * them.
+   *
+   * @param mask The bitmask of solenoids to reserve
+   * @return 0 if successful, mask of solenoids that couldn't be allocated
+   * otherwise
+   */
   virtual int CheckAndReserveSolenoids(int mask) = 0;
 
+  /**
+   * Unreserve the masked solenoids.
```

Comment:
What does the adjective "masked" mean in the context of a solenoid?
I know that this unreserves the solenoids indicated by the bitmask, but it isn't clear.